test/Datadog.Trace.Tests/TracerSettingsTests.cs

```diff
@@ -99,6 +99,19 @@ public void ReplaceLocalhost(string original, string expected)
             Assert.Equal(expected, tracerSettings.AgentUri.ToString());
         }
 
+        [Theory]
+        [InlineData("a,b,c,d,,f", new[] { "a", "b", "c", "d", "f" })]
+        [InlineData(" a, b ,c,,,f ", new[] { "a", "b", "c", "f" })]
```

Comment:
There's no test with only whitespace between commas. I think we can just add it here.
```suggestion
        [InlineData(" a, b ,c, ,,f ", new[] { "a", "b", "c", "f" })]
```